InterviewFlashCardsTests/InterviewFlashCardsTests.m

```
@@ -22,7 +13,6 @@ @implementation InterviewFlashCardsTests
 
 - (void)setUp {
     [super setUp];
-    self.qman = [IFCQueryManager new];
```

@charleshkang Does IFCQueryManager need to be deleted as well?

---

I didn't delete it since it's part of the testing code, but maybe later on if the project is converted to 100% swift?

---

Then why does it appear as tho you have removed it in this PR?

---

Ah, my bad. I wanted to hide the error message that was popping up so I guess I deleted it

---

No worries, it just means if we were to merge this code to master then it will be missing/removed from master. So make sure you go through each line you commit and ensure that it should be included in your current commit/PR. I suggest using `git add -p`, to add files you changed line by line.

---

Gotcha 👍

---

Also don't delete things to get rid of warnings, as obvious as that sounds. Make sure they're actions you want to take and be sure to mention it in your commit if necessary. If it makes you feel better, though, I deleted that test file when I added testing frameworks to the project (work in progress)

---

@mesbekmek since you've already checked this PR would you mind submitting the Review?